reference/index.html

```diff
@@ -2339,7 +2340,7 @@ <h2 id="_overview">Übersicht</h2>
           <div class="sect2">
             <h3 id="_aktuelle_version">Aktuelle Version</h3>
             <div class="paragraph">
-              <p><em>Version</em> : 3.0.13</p>
+              <p><em>Version</em> : 3.0.15</p>
```

Comment:
In https://github.com/europace/baufismart-angebote-api/pull/130 I think we missed something and the committed file was just with blank spaces as "diff". That's why here it's bumping 2 versions at once.